doc/Installation/Installation-Ubuntu-1604-Nginx.md

```diff
@@ -9,9 +9,11 @@ source: Installation/Installation-Ubuntu-1604-Nginx.md
 ```bash
 apt-get install mariadb-server mariadb-client
 systemctl restart mysql
-mysql -uroot -p
+mysql -u root -p
 ```
 
+you should now be presented with the mariadb interactive shell, edit 'password' below to be a secure password, remember this  password as it needs to be used to configure LibreNMS
```

Comment:
My additions are based on a first time installer (me) to give some context and understanding, otherwise there will be installs with the password of password out there - but up to you